lib/brains/forms.py

```diff
@@ -172,43 +169,22 @@ def clean(self):
         return self.cleaned_data
 
 
-class PayMethodUpdateForm(forms.Form):
-    active = forms.BooleanField(required=False)
-
-    def __init__(self, data, obj):
-        self.object = obj
-        return super(PayMethodUpdateForm, self).__init__(data)
+class PayMethodCancelForm(forms.Form):
+    paymethod = PathRelatedFormField(
+        view_name='braintree:mozilla:paymethod-detail',
+        queryset=BraintreePaymentMethod.objects.filter())
 
-    def clean_active(self):
-        active = self.cleaned_data['active']
+    def clean(self):
+        solitude_method = self.cleaned_data.get('paymethod')
+        if not solitude_method:
+            raise forms.ValidationError(
+                'Paymethod is required', code='required'
+            )
 
         # An attempt to enable active on the payment method.
-        if active and not self.object.active:
+        if not solitude_method.active:
             raise forms.ValidationError(
                 'Cannot set an inactive payment method to active',
```

Comment:
This doesn't make sense anymore. If this form is only for removing payment methods, how would it ever activate a payment method?